test/views/UnverifiedEmailSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import utils.FakeRequestCSRFSupport._
+import views.behaviours.ViewBehaviours
+import views.html.unverified_email
+
+class UnverifiedEmailSpec extends DomAssertions with ViewBehaviours {
+
+  lazy val redirectUrl = "/some/url"
+
+  val page = instanceOf[unverified_email]
+
+  val view = () => page(redirectUrl)(fakeRequest.withCSRFToken, messages)
+
+  val messageKeyPrefix = "emailUnverified"
+
+  "Unverified Email Page" must {
+    behave like normalPage(view, messageKeyPrefix, "paragraph1")
+    "have a 'Verify your email address' button with correct link" in {
+      val doc = asDocument(view())
+
+      assertContainsLink(doc, messages(s"${messageKeyPrefix}.link"), redirectUrl)
```

Comment:
Wasn't there a class `ViewMatchers` copied to SFUS? You can then use syntax like `element must haveHref(...)`

Reply:
Didn't realise that. I've updated my test to use the new Matchers class